c/include/nnstreamer-single.h

```diff
@@ -259,6 +259,23 @@ int ml_single_set_property (ml_single_h single, const char *name, const char *va
  */
 int ml_single_get_property (ml_single_h single, const char *name, char **value);
 
+/**
+ * @brief Make a single instance with given ml-option.
+ * @since_tizen 7.0
+ * @remarks %http://tizen.org/privilege/mediastorage is needed if @a model is relevant to media storage.
```

Comment:
line 265 and 266
`... is needed if neural network model is relevant ...`
This function does not have the param 'model ', so need to fix it.